config/config.go

```diff
@@ -236,3 +240,16 @@ func (c *Config) Validate() error {
 	// returns nil if no errors in slice
 	return errors.Join(e...)
 }
+
+var defaultHeadersToExtract = []string{
+	"User-Agent",
+}
+
+// getHTTPHeadersToExtract returns the list of HTTP headers to extract from a HTTP request/response
+func getHTTPHeadersToExtract() []string {
+	httpHeaders := utils.LookupEnvAsStringSlice("HTTP_HEADERS")
+	if len(httpHeaders) > 0 {
```

Comment:
🤔 Does this mean that if I `export HTTP_HEADERS=""` intending for the agent to record _no_ headers, that I will get the defaults anyway?

Reply:
Hmm, yes that is the case and agree that is not the preferred behaviour. I'll update the logic so that if the env var is set with an empty value, it doesn't set any headers.